src/protocol/block.ml

```diff
@@ -106,6 +105,28 @@ let of_yojson json =
 
 let compare a b = BLAKE2B.compare a.hash b.hash
 
+let make ~state_root_hash ~withdrawal_handles_hash ~validators_hash
```

Comment:
Why did you move `make`? I don't see any new dependency on the functions above it.